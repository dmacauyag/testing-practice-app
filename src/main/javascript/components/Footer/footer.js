import React from 'react'

const Footer = () => {
  return (
    <footer>
      <div>I am a footer.</div>
    </footer>
  )
}

export default Footer
