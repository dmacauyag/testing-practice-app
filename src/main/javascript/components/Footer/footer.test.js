import React from 'react'
import Footer from './footer'
import renderer from 'react-test-renderer'

describe('the Footer component', () => {
  it('should correctly render a footer element', () => {
    const component = renderer.create(
      <Footer />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
