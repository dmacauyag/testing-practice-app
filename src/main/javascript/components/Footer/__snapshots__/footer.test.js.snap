// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Footer component should correctly render a footer element 1`] = `
<footer>
  <div>
    I am a footer.
  </div>
</footer>
`;
